### DISCUSSION TITLE
Can local matching be performed

### DISCUSSION THREAD

**Discussion by monkeycc** [kornia/kornia#3098] [Questions]
![微信截图_20241221112020](https://github.com/user-attachments/assets/fbaa745a-bc6b-4cc4-82b9-5c1cd75072ef)

For example, my frame O
You can find all the O's

template matching

![微信截图_20241224154158](https://github.com/user-attachments/assets/26a46e08-76bd-4ab3-825f-ff44a370194d)

**Comment by edgarriba** [comment]
I believe we don't have any template matching algorithm today but would be interesting to have. /cc @ducha-aiki
